dvc/config.py

```diff
@@ -141,7 +141,7 @@ class RelPath(str):
         Optional("hardlink_lock", default=False): Bool,
         Optional("no_scm", default=False): Bool,
         Optional("autostage", default=False): Bool,
-        Optional("experiments", default=False): Bool,
+        Optional("experiments", default=True): Bool,
```

Comment:
I guess no reason to set default whatsoever. Let's put a comment here that this is pretty much backward compatibility.